src/dialogs/config-flow/step-flow-pick-handler.ts

```diff
@@ -141,7 +141,7 @@ class StepFlowPickHandler extends LitElement {
       css`
         div {
           overflow: auto;
-          max-height: 600px;
+          max-height: calc(100vh - 300px);
```

Comment:
Isn't this very limiting on mobile ?

Reply:
This shouldn't have been in this PR 🙈 but, no It just makes sure the scroll container fits on the screen, will fine-tune it a bit.

![image](https://user-images.githubusercontent.com/5662298/74518853-da61bc80-4f14-11ea-9cb7-80f2408bf6c9.png)